compiler/src/dotty/tools/dotc/interpreter/Interpreter.scala

```diff
@@ -138,7 +138,7 @@ class Interpreter(implicit ctx: Context) {
         env(qualifier.symbol)
 
       case SeqLiteral(elems, _) =>
-        Seq(elems.map(elem => interpretTreeImpl(elem, env)))
+        elems.map(elem => interpretTreeImpl(elem, env))
```

Comment:
👍

Reply:
Failed tests because the check file was not updated, but this bug is fixed :)